packages/yourloops/pages/hcp/teams-list.tsx

```diff
@@ -0,0 +1,301 @@
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import * as React from "react";
+import bows from "bows";
+import { useHistory, RouteComponentProps } from "react-router-dom";
+
+import { makeStyles, Theme } from "@material-ui/core/styles";
+
+import Alert from "@material-ui/lab/Alert";
+import AppBar from "@material-ui/core/AppBar";
+import Breadcrumbs from "@material-ui/core/Breadcrumbs";
+import Button from "@material-ui/core/Button";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import Link from "@material-ui/core/Link";
+import Paper from "@material-ui/core/Paper";
+import Toolbar from "@material-ui/core/Toolbar";
+
+import AddIcon from "@material-ui/icons/Add";
+import DeleteIcon from '@material-ui/icons/Delete';
+import EditIcon from '@material-ui/icons/Edit';
+import HomeIcon from "@material-ui/icons/Home";
+import PersonAddIcon from '@material-ui/icons/PersonAdd';
+
+import { Team } from "../../models/team";
+import { t } from "../../lib/language";
+import apiClient from "../../lib/auth/api";
+import { AuthContext } from '../../lib/auth/hook/use-auth';
+
+interface TeamsListPageState {
+  loading: boolean;
+  errorMessage: string | null;
+  teams: Team[];
+}
+interface BarProps {
+  onCreateTeam: (name: string) => Promise<void>
+}
+interface TeamElementProps {
+  team: Team;
+}
+
+const log = bows("TeamsListPage");
+
+const pageBarStyles = makeStyles((theme: Theme) => {
+  /* eslint-disable no-magic-numbers */
+  return {
+    toolBar: {
+      display: "grid",
+      gridTemplateRows: "auto",
+      gridTemplateColumns: "auto auto auto",
+      paddingLeft: theme.spacing(12),
+      paddingRight: theme.spacing(12),
+    },
+    toolBarRight: {
+      display: "flex",
+    },
+    breadcrumbLink: {
+      display: "flex",
+    },
+    homeIcon: {
+      marginRight: "0.5em",
+    },
+    buttonAddTeam: {
+      marginLeft: "auto",
+    },
+  };
+});
+
+const teamPaperStyles = makeStyles((theme: Theme) => {
+  return {
+    paper: {
+      display: "flex",
+      flexDirection: "column",
+    },
+    paperRoot: {
+      padding: "1em",
+    },
+    firstRow: {
+      display: "flex",
+      flexDirection: "row",
+      marginBottom: theme.spacing(2),
+    },
+    secondRow: {
+      display: "flex",
+      flexDirection: "row",
+    },
+    teamName: {
+      minWidth: "8em",
+    },
+    buttonActionFirstRow: {
+      alignSelf: "center",
+      marginRight: "1em",
+    },
+    divActions: {
+      marginLeft: "2em",
+      display: "flex",
+      flexGrow: 1,
+      justifyContent: "flex-start",
+    },
+  };
+});
+
+function AppBarPage(props: BarProps): JSX.Element {
```

Comment:
As you did for patients list, I would probably move the stuff related to the Bar in a dedicated file.

Reply:
Yes a PR will follow